third_party/3/pyspark/sql/functions.pyi

```diff
@@ -159,7 +159,7 @@ def asc_nulls_last(col: ColumnOrName) -> Column: ...
 def ascii(col: ColumnOrName) -> Column: ...
 def asin(col: ColumnOrName) -> Column: ...
 def atan(col: ColumnOrName) -> Column: ...
-def atan2(col: ColumnOrName) -> Column: ...
+def atan2(col1: ColumnOrName, col2: ColumnOrName) -> Column: ...
```

Comment:
The second argument can also accept `int` and `decimal.Decimal`. In general one code argue that [any object that implements `__float__` is acceptable](https://github.com/apache/spark/blob/24e1e41648de58d3437e008b187b84828830e238/python/pyspark/sql/functions.py#L90-L91).